scripts/s3-dist.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script should be run from the repo's root directory (same directory as serverless.yaml)
+# cd fhir-works-on-aws-deployment
+# chmod +x ./scripts/s3-dist.sh && ./scripts/s3-dist.sh <BUCKET_NAME> <BUCKET_REGION>
+
+# Package output directory is assumed to be default (.serverless). Passing -p flag to `sls package` can break this script.
+# Dependencies: npm, aws CLI (configured), sed, jq
+if [ -z "$1" ] || [ -z "$2" ]; then
+    echo Please provide both the prefix and region for the bucket
+    echo For example: "$0" solutions us-east-1
```

Comment:
What does this echo to cmd line? just confused by the `$0`

Reply:
$0 just references the first segment of the command (the script path and name), in case it gets renamed from s3-dist.sh to something else. 
If the script is ran as `$ ./scripts/asdasdasd.sh my-bucket`, then this prints and exits since both args are required for the script to run: 

```
Please provide both the prefix and region for the bucket
For example: asdasdasd.sh solutions us-east-1
```